public/pages/CreateMonitor/containers/CreateMonitor/utils/formikToMonitor.js

```diff
@@ -28,16 +40,18 @@ export function formikToMonitor(values) {
     triggers: [],
     ui_metadata: {
       schedule: uiSchedule,
-      search: uiSearch,
       monitor_type: values.monitor_type,
+      ...monitorUiMetadata(),
     },
   };
 }
 
 export function formikToInputs(values) {
-  switch (values.searchType) {
-    case SEARCH_TYPE.CLUSTER_METRICS:
+  switch (values.monitor_type) {
```

Comment:
is there a concern to use `monitor_type` instead of `searchType`?

Reply:
I think we should be ok using `monitor_type` here. `ClusterMetrics` has its own search and monitor type, so either value could be used get the `switch` to call the appropriate method. Before doc level monitors were added, `formikToInputs` handled the values to either cluster metrics data or otherwise defaulted to the parsing logic used for bucket and query monitors. `formikToSearch` has logic for calling specific methods based on search type as well.